lib/opentelemetry-mapper.js

```diff
@@ -1,23 +1,22 @@
 'use strict'
 
-const { SeverityNumber } = require('@opentelemetry/api-logs') // TODO: optional import
```

Comment:
What is the motivation for removal here?

Reply:
This is the only direct dependency on `@opentelemetry`, with it removed only a release of `otlp-logger` would be needed to update otel dependencies instead of both packages.

With the whole `SEVERITY_NAME_MAP` already in the source below, it does not seem to add value to import these.